src/utils.ts

```diff
@@ -24,19 +29,19 @@ const selectNextRelay = async (
 ): Promise<RelayInfo | undefined> => {
   const { preferredRelays } = getEnvelopingConfig();
 
-  for (const relayInfo of preferredRelays ?? []) {
+  for (const managerData of preferredRelays ?? []) {
     let hubInfo;
 
     try {
-      hubInfo = await httpClient.getChainInfo(relayInfo.url);
+      hubInfo = await httpClient.getChainInfo(managerData.url);
     } catch (error) {
       continue;
```

Comment:
maybe we should print something instead of handling the error silently?

Reply:
surely some logs may be useful here

Reply:
updated

Reply:
I'd log an error in case no hubInfo has been retrieved

Reply:
updated